jsk_perception/src/polygon_array_color_histogram.cpp

```diff
@@ -64,9 +64,9 @@ namespace jsk_perception
   void PolygonArrayColorHistogram::subscribe()
   {
     sub_info_ = pnh_->subscribe("input/info", 1, &PolygonArrayColorHistogram::infoCallback, this);
-    sub_polygon_.subscribe(*pnh_, "input", max_queue_size_);
-    sub_image_.subscribe(*pnh_, "input/image", max_queue_size_);
-    async_ = boost::make_shared<message_filters::Synchronizer<ApproximateSyncPolicy> >(100);
+    sub_polygon_.subscribe(*pnh_, "input", 10);
+    sub_image_.subscribe(*pnh_, "input/image", 10);
+    async_ = boost::make_shared<message_filters::Synchronizer<ApproximateSyncPolicy> >(max_queue_size_);
```

Comment:
The value of queue_size for syncrhonization was 100 before.
So maybe you should also change the default variable of max_queue_size_ to 100. (currently it is 10)